tools/clang_tidy/test/process_fakes.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I don't see an implementation of something like this anywhere else in the engine repo. At some point, I'd guess that we'll want to pull this out to its own package in the engine repo to share among other packages, or upstream something like https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/src/fake_process_manager.dart to https://github.com/google/process.dart

Reply:
Yeah I agree with that! I'd probably say:

1. Pull this into `tools/pkg/fake_process_manager` as soon as we need it somewhere else.
2. Chat with Chris about moving `flutter_tools`' implementation